### PR TITLE
Update readme and CI

### PR DESCRIPTION
**Pull request by perimetral**
Package `node-uuid` is deprecated now. Instead, new package `uuid` with same functionality is used. Also README updated.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "deepstream.io-storage-mariadb",
-  "version": "1.0.4",
+  "version": "1.0.5b",
```
Comment (yasserf) [inline package.json]: what does b stand for? It doesn't go very well with semantic versioning.

**Comment by yasserf** [comment]
It appears that uuid isn't actually used anywhere in the project btw, its just being required (a)

**Comment by PhillippOhlandt** [comment]
@yasserf there is the change https://github.com/deepstreamIO/deepstream.io-storage-mariadb/pull/2/files#diff-3ccc2f730c0039ee4ce9f5c6a51708dfR9

**Comment by yasserf** [comment]
@PhillippOhlandt I saw the change from uuid-node to uuid, I just don't see where uuid is used at all in the connector ( and why, because records don't have them at that layer ). So maintaing uuid versions seems overkill if never used.

https://github.com/deepstreamIO/deepstream.io-storage-mariadb/search?utf8=%E2%9C%93&q=uuid

**Comment by PhillippOhlandt** [comment]
@yasserf You are completely right. I just checked the import and not if it's actually used somewhere.

File: .travis.yml [hunk 2]
```diff
@@ -21,19 +21,19 @@ addons:
     - g++-4.8
 
 node_js:
-  - "6.1"
```
Comment (yasserf) [inline .travis.yml]: we need to run against 6.1 and on osx for artificat creation purposes

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "deepstream.io-storage-mariadb",
-  "version": "1.0.4",
+  "version": "1.0.6b",
```
Comment (yasserf) [inline package.json]: b?

**Pull request merged**